tests/eval_files/trio911.py

```diff
@@ -592,8 +592,14 @@ async def foo_boolops_1():  # error: 0, "exit", Stmt("yield", line+1)
 
 
 # may shortcut after any of the yields
-async def foo_boolops_2():  # error: 0, "exit", Stmt("yield", line+1) # error: 0, "exit", Stmt("yield", line+1)
-    _ = await foo() and (yield) and await foo() and (yield)
+async def foo_boolops_2():  # error: 0, "exit", Stmt("yield", line+4) # error: 0, "exit", Stmt("yield", line+6)
+    # known false positive - but chained yields in bool should be rare
+    _ = (
+        await foo()
+        and (yield)
+        and await foo()
+        and (yield)  # error: 13, "yield", Stmt("yield", line-2, 13)
+    )
```

Comment:
The behaviour difference in 91X boolop handling, i.e. multiple yields with await between them in the same boolop statement will now error ... which really seems like it does not matter at all. But cases that actually matter, i.e. stuff where you're not cramming a ton of stuff into a single line, should be good as before.